ffi/capi/bindings/dart/lib.g.dart

```diff
@@ -613,6 +660,40 @@ final class _SliceUsize extends ffi.Struct {
   }
 }
 
+final class _SliceUtf16 extends ffi.Struct {
+  external ffi.Pointer<ffi.Uint16> _data;
+
+  @ffi.Size()
+  external int _length;
+
+  // This is expensive
+  @override
+  bool operator ==(Object other) {
+    if (other is! _SliceUtf16 || other._length != _length) {
+      return false;
+    }
+
+    for (var i = 0; i < _length; i++) {
+      if (other._data[i] != _data[i]) {
+        return false;
+      }
+    }
+    return true;
+  }
+
+  // This is cheap
+  @override
+  int get hashCode => _length.hashCode;
```

Comment:
Technically, the `hashCode` here could be the `_length` itself. Which is probably what the `hashCode` implementation on `int`s is already doing.

Reply:
I think we should hash, to get uniform distribution

Reply:
Right.. Forgot about that